package-lock.json

```diff
@@ -0,0 +1,400 @@
+{
+  "name": "saml2-js",
+  "version": "2.0.4",
```

Comment:
I believe we've actually decided not to check in `package-lock.json`s for now (to avoid having two sources of truth: `package.json` + `package-lock.json`). But I'll let @jefff chime in